prometheus/rules.yml

```diff
@@ -0,0 +1,11 @@
+groups:
```

Comment:
Alerting rules, identify metrics and conditions that would send an alert.  We can configure all kinds of things here to watch for.